calico/felix/fetcd.py

```diff
@@ -185,52 +294,36 @@ def wait_for_ready(self):
                 gevent.sleep(RETRY_DELAY)
                 continue
 
-    def _reconnect(self, copy_cluster_id=True):
-        etcd_addr = self.config.ETCD_ADDR
-        if ":" in etcd_addr:
-            host, port = etcd_addr.split(":")
-            port = int(port)
-        else:
-            host = etcd_addr
-            port = 4001
-        if self.client and copy_cluster_id:
-            old_cluster_id = self.client.expected_cluster_id
-            _log.info("(Re)connecting to etcd. Old etcd cluster ID was %s.",
-                      old_cluster_id)
-        else:
-            _log.info("(Re)connecting to etcd. No previous cluster ID.")
-            old_cluster_id = None
-        self.client = etcd.Client(host=host, port=port,
-                                  expected_cluster_id=old_cluster_id)
-
-    @actor_message()
-    def watch_etcd(self, update_splitter):
+    def _load_config(self):
```

Comment:
I realise it's not your fault, but the comment on (new) line 283 "We don't log the" just stops in mid-sentence.